tests/cli/rfhub_importer.py

```diff
@@ -0,0 +1,118 @@
+import unittest
+from pathlib import Path
+
+from rfhub2.cli.rfhub_importer import RfhubImporter
+from rfhub2.cli.api_client import Client
+
+
+class RfhubImporterTests(unittest.TestCase):
+
+    def setUp(self) -> None:
+        self.fixture_path = Path.cwd().joinpath('tests', 'acceptance', 'fixtures')
+        self.client = Client('http//localhost:8000', 'rfhub', 'rfhub')
+        self.rfhub_importer = RfhubImporter(self.fixture_path, False, self.client)
+
+    def test_is_library_with_init_should_return_true_on_library_with_init(self):
+        file = self.fixture_path.joinpath('LibWithInit')
+        result = self.rfhub_importer._is_library_with_init(file)
+        self.assertTrue(result, 'method should return true if file is python library with init')
+
+    def test_is_library_with_init_should_return_false_on_library_without_init(self):
+        file = self.fixture_path.joinpath('LibsWithEmptyInit')
+        result = self.rfhub_importer._is_library_with_init(file)
+        self.assertFalse(result, 'method should return true if file is python library without init')
+
+    def test_is_robot_keyword_file_should_return_true_on_library(self):
+        file = self.fixture_path.joinpath('SingleClassLib', 'SingleClassLib.py')
+        result = self.rfhub_importer._is_robot_keyword_file(file)
+        self.assertTrue(result, 'method should return true if file is python library')
+
+    def test_is_robot_keyword_file_should_return_true_on_libdoc(self):
+        file = self.fixture_path.joinpath('test_libdoc_file.xml')
+        result = self.rfhub_importer._is_robot_keyword_file(file)
+        self.assertTrue(result, 'method should return true if file is libdoc file')
+
+    def test_is_library_file_should_return_true_on_resource(self):
+        file = self.fixture_path.joinpath('test_resource.resource')
+        result = self.rfhub_importer._is_robot_keyword_file(file)
+        self.assertTrue(result, 'method should return true if file is python library')

```

Comment:
Message should be updated

Reply:
yeah, together with test name to reflect used method.